mesos/framework/cluster.go

```diff
@@ -19,11 +19,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/elodina/go-mesos-utils"
 	"sort"
 	"strconv"
 	"strings"
 	"sync"
+
+	utils "github.com/elodina/go-mesos-utils"
```

Comment:
Before we merge this into master lets make sure the go-mesos-utils library gets fleshed out from running in the DataStax Enterprise scheduler. @serejja @miry can we set it up this week in Azure? Lets test this branch for mirror maker this way the go kafka client users can launch it